Graph.cpp

```diff
@@ -3,21 +3,31 @@
 void Graph::read_adjacency_list_from_file(const std::string &file_name, const std::string &graph_type) {
```

Comment:
In this case consider renaming the method to read_adjacency_list_and_matrix_from_file

Reply:
I renamed it to read_adjacency_list_and_matrix() so it won't be too long